src/macos-titlebar.ts

```diff
@@ -72,15 +72,17 @@ export function setupMacosTitleBar(window: BrowserWindow): void {
             }
             
             /* Mark the header as a drag handle */
+            .mx_LegacyRoomHeader,
             .mx_RoomHeader {
                 -webkit-app-region: drag;
                 -webkit-user-select: none;
             }
             /* Exclude header interactive elements from being drag handles */
-            .mx_RoomHeader .mx_RoomHeader_avatar,
-            .mx_RoomHeader .mx_E2EIcon,
-            .mx_RoomHeader .mx_RoomTopic,
-            .mx_RoomHeader .mx_AccessibleButton {
+            .mx_RoomHeader .mx_DecoratedRoomAvatar,
```

Comment:
The text container for the room name in the header should also be no-drag, like it is on legacy (its within the Accessible Button)

Reply:
I'm not entirely sure it needs to be like that. 
The new room header is not interactive the same way the old one way, therefore we can allow drag from that element.

In the spirit of moving that pull request forward i've pushed that change.

Reply:
It'd mean you can't highlight the text which is not good for accessibility